functests-render-command/1_render_command/render_test.go

```diff
@@ -16,55 +16,109 @@ var (
 	assetsInDir  string
 	ppInFiles    string
 	testDataPath string
+	refPath      string
+	cmdLineArgs  []string
+	envVars      []string
 )
 
 var _ = Describe("render command e2e test", func() {
-
 	BeforeEach(func() {
 		assetsOutDir = createTempAssetsDir()
 		assetsInDir = filepath.Join(workspaceDir, "build", "assets")
 		ppInFiles = filepath.Join(workspaceDir, "cluster-setup", "manual-cluster", "performance", "performance_profile.yaml")
 		testDataPath = filepath.Join(workspaceDir, "testdata")
+		refPath = filepath.Join(testDataPath, "render-expected-output")
+
+		cmdLineArgs = []string{
+			filepath.Join(binPath, "performance-addon-operators"),
+			"render",
+		}
+		envVars = []string{}
 	})
 
-	Context("With a single performance-profile", func() {
-		It("Gets cli args and produces the expected components to output directory", func() {
+	JustBeforeEach(func() {
+		fmt.Fprintf(GinkgoWriter, "running: %v\n", cmdLineArgs)
+		cmd := exec.Command(cmdLineArgs[0], cmdLineArgs[1:]...)
+		cmd.Env = append(cmd.Env, envVars...)
+		_, err := cmd.Output()
+		Expect(err).ToNot(HaveOccurred())
+	})
 
-			cmdline := []string{
-				filepath.Join(binPath, "performance-addon-operators"),
-				"render",
+	Context("with a single performance-profile and command line args", func() {
+		BeforeEach(func() {
+			cmdLineArgs = append(cmdLineArgs,
 				"--performance-profile-input-files", ppInFiles,
 				"--asset-input-dir", assetsInDir,
 				"--asset-output-dir", assetsOutDir,
-			}
-			fmt.Fprintf(GinkgoWriter, "running: %v\n", cmdline)
+			)
+		})
+
+		It("should produces the expected components to output directory", func() {
+			file, diff, err := cmpRenderToRef()
```

Comment:
s/cmpRenderToRef/compareRenderToreference 

In general I would rather have longer name with clear meaning